src/ImageSharp/Formats/Tiff/Writers/TiffStreamWriter.cs

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using System.IO;
+using SixLabors.ImageSharp.Memory;
+
+namespace SixLabors.ImageSharp.Formats.Experimental.Tiff.Writers
+{
+    /// <summary>
+    /// Utility class for writing TIFF data to a <see cref="Stream"/>.
+    /// </summary>
+    internal class TiffStreamWriter : IDisposable
+    {
+        private static readonly byte[] PaddingBytes = new byte[4];
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TiffStreamWriter"/> class.
+        /// </summary>
+        /// <param name="output">The output stream.</param>
+        public TiffStreamWriter(Stream output) => this.BaseStream = output;
+
+        /// <summary>
+        /// Gets a value indicating whether the architecture is little-endian.
+        /// </summary>
+        public bool IsLittleEndian => BitConverter.IsLittleEndian;
+
+        /// <summary>
+        /// Gets the current position within the stream.
+        /// </summary>
+        public long Position => this.BaseStream.Position;
+
+        public Stream BaseStream { get; }
+
+        protected MemoryAllocator MemoryAllocator { get; }
```

Comment:
MemoryAllocator and Configuration below seems not to be used. Can they be removed?

Reply:
sure, done!